curriculum/challenges/english/01-responsive-web-design/learn-css-colors-by-building-a-color-markers-set/step-089.md

```diff
@@ -0,0 +1,112 @@
+---
+id: 61b30a286c228bd0c493c09a
+title: Step 89
+challengeType: 0
+dashedName: step-89
+---
+
+# --description--
+
+But what if you wanted to expand the shadow out further? You can do that with the optional `spreadRadius` value:
+
+```css
+box-shadow: offsetX offsetY blurRadius spreadRadius color;
+```
+
+Like `blurRadius`, `spreadRadius` defaults to `0` if it isn't included.
+
+Practice by adding a 5 pixel shadow directly around the blue marker.
+
+In the `.blue` CSS rule, add the `box-shadow` property with the values `0` for `offsetX`, `0` for `offsetY`, `0`, for `blurRadius`, `5px` for `spreadRadius`, and `blue` for `color`.
+
+# --hints--
+
+Your `.blue` CSS rule should have a `box-shadow` shorthand property and with the value `0 0 0 5px blue`.
+
+```js
+console.log(new __helpers.CSSHelp(document).getStyle('.blue'));
```

Comment:
```suggestion
```

Reply:
Which do you dislike more, `console.log` or jQuery? 🙃

Reply:
Hardly a question. I use `console.log`, do I not. I am not sure what the replacement would be....

Reply:
Darn it, I knew I must have missed one of these. Thanks for catching this @ShaunSHamilton :+1: